ansys/dpf/core/core.py

```diff
@@ -517,10 +517,10 @@ def download_files_in_folder(
 
         Parameters
         ----------
-        server_folder_path : str
+        server_folder_path : str or os.PathLike
```

Comment:
@PProfizi could you please download the built doc out of the github actions and make sure that the API documentation of what you changed looks good? Because I'm afraid the "or" will not, since we always use ","

Reply:
@cbellot000 
Ok I'll change that. I started using "or" because a lot of times there was already "str, optional" and adding a coma seemed more confusing to me (like: "str, os.PathLike, optional").

Reply:
@cbellot000 I found an example of docstring similar to what I did here:

> fields_container (ansys.grpc.dpf.collection_pb2.Collection or FieldsContainer, optional)


in 
> class ansys.dpf.core.custom_fields_container.ElShapeFieldsContainer(fields_container=None, server=None)